invenio_communities/assets/semantic-ui/js/invenio_communities/components/Truncate.js

```diff
@@ -0,0 +1,36 @@
+import PropTypes from 'prop-types';
+import React, { Component } from 'react';
+
+/**
+ * Applying an ellipsis to multiline text
+ * https://stackoverflow.com/questions/33058004/applying-an-ellipsis-to-multiline-text
+ */
+export class Truncate extends Component {
+  render() {
+    const { lines, width, children } = this.props;
+    const multilineEllipsisStyle = {
+      overflow: 'hidden',
+      textOverflow: 'ellipsis',
+      display: '-webkit-box',
+      WebkitLineClamp: lines,
+      whiteSpace: 'normal',
+      WebkitBoxOrient: 'vertical',
+    };
+    if (width) {
+      multilineEllipsisStyle['maxWidth'] = width;
```

Comment:
what is the unit of width ? what about responsiveness? if width is set as fixed, I believe it will break depending on resolution